plugins/shrink-path/README.md

```diff
@@ -37,6 +38,7 @@ The following options are available:
 
 ```
     -f, --fish       fish simulation, equivalent to -l -s -t.
+    -g, --global     Add asterix to allow globbing of shrunk path.
```

Comment:
This appears to be a typo.
```suggestion
    -g, --glob     Add asterix to allow globbing of shrunk path.
```

Reply:
Thanks for catching that!  I have committed your suggestion.